src/System.Private.CoreLib/shared/System/MathF.cs

```diff
@@ -183,18 +183,37 @@ public static unsafe float Round(float x, int digits, MidpointRounding mode)
 
                 x *= power10;
 
-                if (mode == MidpointRounding.AwayFromZero)
+                if (mode is MidpointRounding.ToEven)
```

Comment:
would a switch statement be clearer?

Reply:
We should need to select a mode (perhaps none) as the default case

Reply:
I've updated the implementation in Math and MathF with a switch statement